Packs/RSS/Integrations/RSS/RSS.py

```diff
@@ -0,0 +1,62 @@
+import email.utils
+import json
+import ssl
+from datetime import datetime
+from time import mktime
+
+import demistomock as demisto  # noqa: F401
+import feedparser
+from CommonServerPython import *  # noqa: F401
+
+
+def fetch_incidents(url):
+
+    feed = feedparser.parse(url)
+    incidents = []
+
+    # demisto.getLastRun() will returns an obj with the previous run in it.
+    last_run = demisto.getLastRun()
+    # Get the last fetch time, if exists
+    last_fetch = last_run.get('last_fetch')
+
+    if last_fetch == None:
+        last_fetch = datetime(1970, 1, 1)
+    else:
+        last_fetch = datetime.strptime(last_fetch, '%Y-%m-%dT%H:%M:%S.%f')
+
+    for entry in feed.entries:
+
+        date_parsed = email.utils.parsedate(entry.published)
+        dt = datetime.fromtimestamp(mktime(date_parsed))
+
+        incident = {
+            'name': entry.title,
+            'occured': dt.isoformat(),
+            'rawJSON': json.dumps(entry)
+        }
+
+        incidents.append(incident)
+
+    dtnow = datetime.now().strftime('%Y-%m-%dT%H:%M:%S.%f')
+    demisto.setLastRun({'last_fetch': dtnow})
+
+    return incidents
+
+
+##
+try:
+    if demisto.command() == 'test-fetch-rss':
+        print(fetch_incidents(demisto.params().get('URL')))
```

Comment:
Please do not use Print in the code, it is better to use return_result()
```suggestion
        return_results(fetch_incidents(demisto.params().get('URL')))
```